runtime/src/lib.rs

```diff
@@ -138,12 +138,12 @@ impl_opaque_keys! {
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("data-avail"),
 	impl_name: create_runtime_str!("data-avail"),
-	authoring_version: 10,
+	authoring_version: 11,
 	// Per convention: if the runtime behavior changes, increment spec_version
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 6,
+	spec_version: 7,
```

Comment:
It seems the Runtime has no changed, so we have to keep the `spec_version`.
Internal changes like dependency update would require bumping up of `impl_version`